client/tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This file is super fragile, if you touch it, you'll need to test a bunch of stuff and look for VSCode problems